scrapy/templates/spiders/csvfeed.tmpl

```diff
@@ -2,19 +2,19 @@ from scrapy.spiders import CSVFeedSpider
 
 
 class $classname(CSVFeedSpider):
-    name = '$name'
-    allowed_domains = ['$domain']
-    start_urls = ['http://$domain/feed.csv']
-    # headers = ['id', 'name', 'description', 'image_link']
-    # delimiter = '\t'
+    name = "$name"
+    allowed_domains = ["$domain"]
+    start_urls = ["http://$domain/feed.csv"]
+    # headers = ["id", "name", "description", "image_link"]
+    # delimiter = "\t"
 
     # Do any adaptations you need here
     #def adapt_response(self, response):
-    #    return response
+    #     return response
```

Comment:
This indentation change seems wrong (4 spaces before, 5 now), but not a big issue, i.e. I don’t mind merging anyway.

Reply:
You know what, let me address it quickly, and let’s include this change into Scrapy 2.8.